### ISSUE TITLE
Certain ARIA roles must contain particular children

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#3834]
## Summary

When using certain roles, we must have specific children that satisfy the requirements for the role being used.

## Locations

https://github.com/GCTC-NTGC/gc-digital-talent/blob/759d780c3dbde4012aac0b96515f1b5dacb0334d/frontend/admin/src/js/components/Table/Table.tsx#L250

https://github.com/GCTC-NTGC/gc-digital-talent/blob/759d780c3dbde4012aac0b96515f1b5dacb0334d/frontend/common/src/components/skills/SkillPicker/SkillPicker.tsx#L303-L317

https://github.com/GCTC-NTGC/gc-digital-talent/blob/c8ff1a196d4275a58269f4273774e165366ee676/frontend/common/src/components/SearchRequestFilters/SearchRequestFilters.tsx#L228-L251

**Comment by esizer** [comment]
I believe we have completed 2/3 for this one. The only remaining one should be the `Table.tsx` item.

## Chip

https://github.com/GCTC-NTGC/gc-digital-talent/blob/70518649212b177e78866304c9eb6f134ba9c908/frontend/common/src/components/Chip/Chips.tsx#L9

https://github.com/GCTC-NTGC/gc-digital-talent/blob/70518649212b177e78866304c9eb6f134ba9c908/frontend/common/src/components/Chip/Chip.tsx#L65

**Comment by mnigh** [comment]
@esizer is this issue still relevant? If so, can the description be updated with an acceptance criteria and a proposed solution (if you have one)?
